### ISSUE TITLE
Using only `sort_by` returns a 500

### ISSUE THREAD

**Issue by Seluj78** [shellhub-io/shellhub#3498]
### Description

When requesting to get devices using `GET /api/devices` with query params `?sort_by=name`, you get a 500 returned because a `order_by` wasn't supplied.

I think `order_by` should have a default value to avoid unpleasant surprises like these

Reproduction:

```python
# 500
print(requests.get("https://shellhub.example.com/api/devices?sort_by=name", headers={"Authorization": f"Bearer {token}"}).status_code)

# 200
print(requests.get("https://shellhub.example.com/api/devices", headers={"Authorization": f"Bearer {token}"}).status_code)

# 200
print(requests.get("https://shellhub.example.com/api/devices?sort_by=name&order_by=asc", headers={"Authorization": f"Bearer {token}"}).status_code)
```

### Edition

Community

### Version

0.14.1

**Comment by heiytor** [comment]
Hi, @Seluj78

This has already been addressed in a44c7608f and will be included in the upcoming release.

**Comment by Seluj78** [comment]
Great thank you ! I'll let you close the issue if you want or keep it open for tracking :)

**Comment by heiytor** [comment]
I will keep this issue open until the release, allowing you to test it yourself and verify whether the issue has been resolved.

**Comment by Seluj78** [comment]
Sounds good !

**Comment by gustavosbarreto** [comment]
We have just released a release candidate `v0.14.3-rc.1` with the latest fixes

**Comment by Seluj78** [comment]
Can confirm `v0.14.3-rc.1` has fixed this issue ! I will let you close it as completed
